src/test/java/test/integration/extension/AddDependencies.java

```diff
@@ -0,0 +1,20 @@
+package test.integration.extension;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Defines set of dependencies which will be added to deployed test archive.
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+public @interface AddDependencies {
+
+    /**
+     * List of maven artifact coordinates in the format of groupId:artifactId:version.
+     * @return
+     */
+    String[] value();
```

Comment:
About splitting I would say that it is the same case of `@Environment` case on Cube. WE could do @Environment("A=B") but user can do an error so we changed to `@Environment(key="", value="")`